payments-core/src/main/java/com/stripe/android/model/LinkAccountSession.kt

```diff
@@ -0,0 +1,10 @@
+package com.stripe.android.model
+
+import com.stripe.android.core.model.StripeModel
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+internal data class LinkAccountSession(
```

Comment:
Could it be something like `BankConnectionLinkAccountSession` to avoid confusion with Link the other product?

Reply:
Maybe `BankAccountLinkingSession`

Reply:
the official name of this is `BankConnectionsLinkAccountSession`, how does that sound? (https://git.corp.stripe.com/stripe-internal/pay-server/blob/master/lib/bank_connections/auth_session/data/link_account_session.rb)

Reply:
Renamed using that prefix - https://github.com/stripe/stripe-android/pull/4711/commits/5e1711c79a035e674ec130b0601e3b040513f412

Reply:
Thanks!